### DISCUSSION TITLE
Community edition

### DISCUSSION THREAD

**Discussion by devistar** [amplication/amplication#5986] [Q&A] [answered]
Is it possible to self host the community edition for commercial use ?
What are the limitations compared to the enterprise version?

**Comment by eric-gitta-moore** [comment]
Any news ?

**Comment by eric-gitta-moore** [comment]
After the community version is compiled into a Docker image, does it provide a Docker Compose one click start?

**Reply by yuval-hazaz** [reply]
The platform is deployed with several services and there is no single container for deployment

**Comment by yuval-hazaz** [comment, marked as answer]
hi, all the details about licensing can be found here
https://github.com/amplication/amplication/blob/master/LICENSE

In general:
* All content that resides under the "ee/" directory of this repository, if that directory exists, is licensed under the license defined in "ee/LICENSE".
* Content outside of the above mentioned directories or restrictions above is available under the "Apache License 2.0" as defined below.
